### ISSUE TITLE
tasks are not registered

### ISSUE THREAD

**Issue by buus2** [neptune-ml/minerva#47]
For
```
python main.py -- submit --problem whales --task_nr 1 --filepath resources/whales/tasks/task1.ipynb
```
I obtain
```
Traceback (most recent call last):
  File "main.py", line 72, in <module>
    action()
  File "/home/patryk/Documents/edukacyjne/Minerva/0401/minerva_venv/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/patryk/Documents/edukacyjne/Minerva/0401/minerva_venv/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/patryk/Documents/edukacyjne/Minerva/0401/minerva_venv/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/patryk/Documents/edukacyjne/Minerva/0401/minerva_venv/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/patryk/Documents/edukacyjne/Minerva/0401/minerva_venv/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "main.py", line 67, in submit
    pm.submit_task(task_subproblem, task_nr, filepath, dev_mode, cloud_mode)
  File "/home/patryk/Documents/edukacyjne/Minerva/0401/minerva/minerva/whales/problem_manager.py", line 42, in submit_task
    task_handler = registered_tasks[task_nr](trainer)
KeyError: 1
```
I checked that the error raises because `registered_tasks` is `{}`, but this is all I understand. I encountered the same issue for fashion_mnist problem.

**Comment by jakubczakon** [comment]
Done, waiting for PR to merge.

**Comment by kamil-kaczmarek** [comment]
@jakubczakon @buus2 merged #48
